cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -408,6 +411,54 @@ func resolveRevision(root *url.URL, repo, spec string) (string, error) {
 	return b.String(), nil
 }
 
+func ping(root *url.URL) error {
+	u := root.ResolveReference(&url.URL{Path: "/.internal/ping"})
+	resp, err := http.Get(u.String())
+	if err != nil {
+		return err
+	}
+
+	defer resp.Body.Close()
+	if resp.StatusCode != http.StatusOK {
+		return fmt.Errorf("ping: bad HTTP response status %d", resp.StatusCode)
+	}
+	body, err := ioutil.ReadAll(io.LimitReader(resp.Body, 1024))
```

Comment:
Could we set `1024` to `15` to avoid doing the truncating logic below?

Reply:
yes it would! will update.